src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBaseKernels.hpp

```diff
@@ -698,7 +698,7 @@ class ElementBasedAssemblyKernel
    * @return the ghost rank of the element
    */
   GEOSX_HOST_DEVICE
-  integer getElemGhostRank( localIndex const ei ) const
+  integer elemGhostRank( localIndex const ei ) const
```

Comment:
can we just get rid of this accessor and use the member?

Reply:
Yes, we can get rid of that. I am taking note of this and will remove in the next PR.